erpnext/hr/doctype/employee_loan/employee_loan.js

```diff
@@ -36,7 +36,21 @@ frappe.ui.form.on('Employee Loan', {
 		})
 	},
 
-	mode_of_payment: function(frm){
+	on_submit: function(frm) {
+		if (frm.status!="Loan Unpaid") {
+			return frappe.call({
+				doc: cur_frm.doc,
```

Comment:
Always avoid passing doc. In the server side, make the function whitelisted, pass required args from client.